src/backend/core/api/serializers.py

```diff
@@ -120,11 +167,33 @@ def validate(self, attrs):
         return attrs
 
 
+class TeamAccessReadOnlySerializer(TeamAccessSerializer):
+    """Serialize team accesses for list and retrieve actions."""
+
+    abilities = serializers.SerializerMethodField(read_only=True)
+    user = UserSerializer(read_only=True)
+
+    class Meta:
+        model = models.TeamAccess
+        fields = [
+            "id",
+            "user",
+            "role",
+            "abilities",
+        ]
+        read_only_fields = [
+            "id",
+            "user",
+            "role",
+            "abilities",
+        ]
+
+
 class TeamSerializer(serializers.ModelSerializer):
     """Serialize teams."""
 
     abilities = serializers.SerializerMethodField(read_only=True)
-    accesses = TeamAccessSerializer(many=True, read_only=True)
+    # accesses = TeamAccessSerializer(many=True, read_only=True)
```

Comment:
Remove accesses field returned upon calling `/teams/<id>/` view. Accesses are now available from `/teams/<id>/accesses/`.

Reply:
Yes, I kinda wanted to make sure there was no secret reason to keep this field. But if you don't see any, let's remove it and we'll add it again later, if needed.